addons/pos_restaurant/static/src/app/control_buttons/take_away_button/take_away_button.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module */
+
+import { usePos } from "@point_of_sale/app/store/pos_hook";
+import { ProductScreen } from "@point_of_sale/app/screens/product_screen/product_screen";
+import { Component } from "@odoo/owl";
+
+export class takeAwayButton extends Component {
```

Comment:
```suggestion
export class TakeAwayButton extends Component {
```

Use PascalCase when naming a component. Also rename the template.